packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -0,0 +1,401 @@
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {NewIcon} from '@twilio-paste/icons/esm/NewIcon';
+import {ProductVoiceIcon} from '@twilio-paste/icons/esm/ProductVoiceIcon';
+import {ProductElasticSIPTrunkingIcon} from '@twilio-paste/icons/esm/ProductElasticSIPTrunkingIcon';
+import {Grid, Column} from '@twilio-paste/grid';
+
+import {Truncate} from '@twilio-paste/truncate';
+import {styled, css} from '@twilio-paste/styling-library';
+
+import {Box} from '@twilio-paste/box';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Stack} from '@twilio-paste/stack';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {useUID} from '@twilio-paste/uid-library';
+
+import {SidebarCategoryRoutes} from '../../../constants';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {tableExample, betaFeatureExample, settingsAndProducts} from '../../../component-examples/BadgeExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/badge"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/badge/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Badge"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Badge"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/badge"
+  storybookUrl="/?path=/story/components-badge--all-badges"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Badge
+
+A Badge is a small chunk of information that describes an object in a visually highlighted way. You can think of badges as a short description of a more complex object that is highlighted to draw visual focus. Badges should be used to describe static attributes. Badges should be used to highlight static attributes of an object, for example to identify a feature that is available in a `beta` release phase. For dynamic attributes, such as the status of a dynamic process, refer to the [Status Pattern](https://paste.twilio.design/roadmap 'See our roadmap for the Status Pattern release date').
```

Comment:
> Badges should be used to describe static attributes. Badges should be used to highlight static attributes of an object, 

Is it just me or does this read a little off? Maybe that first sentence is redundant?

Reply:
Totally, this is just what I drafted. Would love to get some input from @kelseycavitt, @loreina, and @katieporter.

Reply:
I think we might look to soften this copy a little. It reads very technical, and maybe a little too formal.

as an example if you look at the Atlassian equivalent tags, it reads a little more casually https://atlassian.design/components/tag/usage